engine/src/main/java/org/pentaho/di/repository/Repository.java

```diff
@@ -759,7 +759,7 @@ public void saveDatabaseMetaJobEntryAttribute( ObjectId id_job, ObjectId id_jobe
    * @return the Metastore that is implemented in this Repository. Return null if this repository doesn't implement a
    *         Metastore.
    */
-  public IMetaStore getMetaStore();
+  public IMetaStore getRepositoryMetaStore();
```

Comment:
I see lot of places we are changing the name from getMetastore() to getRepositoryMetastore(), not sure why?

Reply:
This change is to make the metastore flow clearer, the getRepositoryMetastore method should only be used when we are actively trying to get an instance of the Repository Metastore specifically. I have changed all other calls where it makes sense to get the default metastore.

Reply:
Earlier there was no distinction between the two. Are we are calling it with two different names because they can be out of sync

Reply:
Yes exactly, pretty much the only place that should be calling getRepositoryMetastore should be places like export/import from repository specifically or the RepositoryProvider